### PR TITLE
Feature/telephony group update form to new oui-form

### PR DESCRIPTION
**Pull request by JDownloader**

### CODE REVIEW

File: client/app/telecom/telephony/billingAccount/administration/optionsGroup/telecom-telephony-billing-account-administration-options-group.html [hunk 1]
```diff
@@ -16,134 +9,57 @@ <h1 data-translate="telephony_options_group_title"></h1>
...
-                                           data-uib-btn-radio="false"
-                                           data-translate="common_no_capitalized">
-                                    </label>
-                                </div>
-                            </div>
-                        </div>
-                    </div> <!-- END OF HIDDEN EXTERNAL NUMBER -->
-                    <!-- /.widget-presentation -->
+        <oui-back-button data-title="{{:: 'telephony_options_group_title' | translate }}"
+                         data-on-click="OptionsGroupCtrl.$state.go('telecom.telephony.administration')">
```
Comment (Jisay) [inline client/app/telecom/telephony/billingAccount/administration/optionsGroup/telecom-telephony-billing-account-administration-options-group.html]: this is not a button but a link. An href must be used ;-)

File: client/app/telecom/telephony/billingAccount/administration/optionsGroup/translations/Messages_fr_FR.xml [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
+   <translation id="telephony_billing_account_administration_options_group_overrideDisplayedNumber_info">Présenter le numéro du standard pour les appels vers l'extérieur et votre propre numéro pour les appels internes.</translation>
+   <translation id="telephony_billing_account_administration_options_group_overrideDisplayedNumber_label" qtlid="456856">Utiliser un affichage distinct</translation>
    <translation id="telephony_billing_account_administration_options_group_displayOnIncomingCall" qtlid="456869">Affichage de la description des lignes</translation>
    <translation id="telephony_billing_account_administration_options_group_displayOnIncomingCall_info" qtlid="456882">Présenter la description des lignes lors des appels.</translation>
-   <translation id="telephony_billing_account_administration_options_group_displayOnIncomingCall_label" qtlid="456895">Présentation de la description lors des appels&#160;</translation>
+   <translation id="telephony_billing_account_administration_options_group_displayOnIncomingCall_label">Présentation de la description lors des appel</translation>
```
Comment (Jisay) [inline client/app/telecom/telephony/billingAccount/administration/optionsGroup/translations/Messages_fr_FR.xml]: "appels" ?

File: client/app/telecom/telephony/billingAccount/administration/optionsGroup/telecom-telephony-billing-account-administration-options-group.html [hunk 3]
```diff
@@ -16,134 +9,57 @@ <h1 data-translate="telephony_options_group_title"></h1>
...
-                                           data-uib-btn-radio="false"
-                                           data-translate="common_no_capitalized">
-                                    </label>
-                                </div>
-                            </div>
-                        </div>
-                    </div> <!-- END OF OVERRIDE DISPLAYED NUMBER -->
-                    <!-- /.widget-presentation -->
+        <form name="OptionsGroupCtrl.optionsGroupSettings"
+              data-ng-submit="OptionsGroupCtrl.changeSettings()">
```
Comment (tremblaymath) [inline client/app/telecom/telephony/billingAccount/administration/optionsGroup/telecom-telephony-billing-account-administration-options-group.html]: I know there is no validation in this form but to standardize and create a clean copy/paste example, you should check if form is $valid before submitting. I saw self.optionsGroupSettings.$invalid in the controller but it's not standard with other integrations

File: client/app/telecom/telephony/billingAccount/administration/optionsGroup/telecom-telephony-billing-account-administration-options-group.html [hunk 4]
```diff
@@ -16,134 +9,57 @@ <h1 data-translate="telephony_options_group_title"></h1>
...
-                                           data-uib-btn-radio="false"
-                                           data-translate="common_no_capitalized">
-                                    </label>
-                                </div>
-                            </div>
-                        </div>
-                    </div> <!-- END OF DISPLAY ON INCOMING CALL -->
-                    <!-- /.widget-presentation -->
+            <oui-field data-label="{{:: 'telephony_billing_account_administration_options_group_hiddenExternalNumber_label' | translate }}">
+                <oui-radio name="hiddenExternalNumber"
```
Comment (tremblaymath) [inline client/app/telecom/telephony/billingAccount/administration/optionsGroup/telecom-telephony-billing-account-administration-options-group.html]: You should wrap oui-radio in a oui-radio-group http://master.ui-kit.ovh/#!/oui-angular/radio-group

File: client/app/telecom/telephony/billingAccount/administration/optionsGroup/translations/Messages_fr_FR.xml [hunk 5]
```diff
@@ -2,13 +2,13 @@
...
+   <translation id="telephony_billing_account_administration_options_group_hiddenExternalNumber_label">Masquer les appels entrants / sortants</translation>
    <translation id="telephony_billing_account_administration_options_group_overrideDisplayedNumber" qtlid="456830">Présentation numéro interne/externe</translation>
-   <translation id="telephony_billing_account_administration_options_group_overrideDisplayedNumber_info" qtlid="456843">Trés utile dans une entreprise. Présentez le numéro du standard pour vos appels extérieurs et présentez votre propre numéro pour les appels internes à l’entreprise.</translation>
-   <translation id="telephony_billing_account_administration_options_group_overrideDisplayedNumber_label" qtlid="456856">Utiliser un affichage distinct&#160;:</translation>
+   <translation id="telephony_billing_account_administration_options_group_overrideDisplayedNumber_info">Présenter le numéro du standard pour les appels vers l'extérieur et votre propre numéro pour les appels internes.</translation>
+   <translation id="telephony_billing_account_administration_options_group_overrideDisplayedNumber_label" qtlid="456856">Utiliser un affichage distinct</translation>
```
Comment (tremblaymath) [inline client/app/telecom/telephony/billingAccount/administration/optionsGroup/translations/Messages_fr_FR.xml]: You should remove the qtlid else the "&#160;" will not be removed for english and other languages

File: client/app/telecom/telephony/billingAccount/administration/optionsGroup/telecom-telephony-billing-account-administration-options-group.html (hunk 4 above)
Comment (JDownloader) [reply client/app/telecom/telephony/billingAccount/administration/optionsGroup/telecom-telephony-billing-account-administration-options-group.html]: This almost a copy/paste from http://master.ui-kit.ovh/#!/oui-angular/field

**Pull request merged**